.vsconfig

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If I need to delete this file lemme know, it keeps trying to add it to my commits. I'm using Visual Studio Community 2019.

Reply:
Ya, no need for the vsconfig